lms/djangoapps/shoppingcart/tests/payment_fake.py

```diff
@@ -202,9 +202,6 @@ def _payment_page_response(self, post_params):
         Render the payment page to a response.  This is an HTML form
         that triggers a POST request to `callback_url`.
 
-        The POST params are described in the CyberSource documentation:
-        http://apps.cybersource.com/library/documentation/dev_guides/HOP_UG/html/wwhelp/wwhimpl/js/html/wwhelp.htm
```

Comment:
Instead of deleting this entirely, I'd suggest replacing the old documentation link with the new documentation link: http://apps.cybersource.com/library/documentation/dev_guides/Secure_Acceptance_WM/Secure_Acceptance_WM.pdf.